Code/Common/include/sitkInterpolator.h

```diff
@@ -80,6 +80,13 @@ enum InterpolatorEnum {
    */
   sitkLabelGaussian = 5,
 
+  /** \brief Smoothly interpolate multi-label images
+   *
+  /** \brief Linear interpolation for multi-label images
```

Comment:
There is a mistake in the comments with two opening "/**"

Reply:
fixed